Modules/Project/Resources/views/subviews/edit-project-team-members.blade.php

```diff
@@ -0,0 +1,53 @@
+<div class="card">
+    <div class="card-header" data-toggle="collapse" data-target="#project_team_member_form">
+        <h4>Project team members</h4>
+    </div>
+
+    <div id="project_team_member_form" class="collapse hide">
+        <form  action="{{ route('project.update', $project) }}" method="POST" id="update_project_team_member_form">
+            @csrf
+            <input type="hidden" value="project_team_members" name="update_section"> 
+
+            <div class="card-body">
+                <div class="row">
+                    <div class="col-4">
+                        Team Members
+                    </div>
+                    <div class="col-4">
+                        Designations
+                    </div>
+                </div>
+
+                <div class="row mb-3" v-for="(projectTeamMember, index) in projectTeamMembers" :key="projectTeamMember.id">
+                    <div class="col-4">
+                        <select v-model="projectTeamMember.id" :name="`projectTeamMember[${index}][team_member_id]`" class="form-control">
+                            <option value="">Select team member</option>
+                            <option v-for="(teamMember) in allTeamMembers" :value="teamMember.id" :key="teamMember.id">@{{ teamMember.name }}</option>
+                        </select>
+                    </div>
+                    <div class="col-4">
+                        <select v-model="projectTeamMember.pivot.designation" :name="`projectTeamMember[${index}][designation]`" class="form-control">
+                            <option value="">Select Designations</option>
+
+                            <option v-for="(teamMemberDesignation, key) in teamMembersDesignations" :value="key">@{{ teamMembersDesignation }}</option>
+                        </select>
+                    </div>
+
+                    <div class="col-4">
+                        <button v-on:click="removeProjectTeamMember(index)" type="button" class="btn btn-danger btn-sm mt-1 ml-2"> - </button>
```

Comment:
can we do this? May look a bit nicer on UI
```suggestion
                        <button v-on:click="removeProjectTeamMember(index)" type="button" class="btn btn-danger btn-sm mt-1 ml-2 text-white fz-14">Remove</button>
```